js/ui/components/clubhouse.js

```diff
@@ -290,6 +290,33 @@ var ClubhouseNotificationBanner = new Lang.Class({
                 button.destroy();
         });
     },
+
+    _slideOut: function() {
+        let monitor = Main.layoutManager.primaryMonitor;
+        if (!monitor)
+            return;
+
+        let endX = monitor.x + monitor.width;
+
+        Tweener.addTween(this.actor,
+                         { x: endX,
+                           time: CLUBHOUSE_BANNER_ANIMATION_TIME,
+                           transition: 'easeOutQuad',
+                           onComplete: () => {
+                               this.actor.destroy();
```

Comment:
Perhaps call `this.destroy(false)` here? Otherwise set `this.actor = null` as well for consistency.

Reply:
Hmm, `this.destroy` is the function that calls this. That's inviting unwanted/erroneous recursion down the road.
I should however let the actor be null as well in the `this.destroy` function even when calling the slideOut.